build/scripts/locate-vs.ps1

```diff
@@ -13,7 +13,7 @@ function Create-Directory([string[]] $path) {
 
 function Locate-LocateVsApi {
   $packagesPath = Locate-PackagesPath
-  $locateVsApi = Join-Path -path $packagesPath -ChildPath "RoslynTools.Microsoft.LocateVS\$locateVsApiVersion\lib\net46\LocateVS.dll"
```

Comment:
The Tool version includes all dependencies alongside, where-as the Library version brings in dependencies properly (via NuGet Dependency list)